src/pages/admin/EventDetails.js

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState} from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import { useParams, useHistory, withRouter } from "react-router-dom";
+import { connect } from "react-redux";
+import { Helmet } from "react-helmet";
+import CircularProgress from "@material-ui/core/CircularProgress";
+import ThemeProvider from "../../components/ThemeProvider";
+import { updateEvents, updateRegistrations } from "../../utils";
+import EventDescription from "../../components/EventDescription";
+import ArrowBackIcon from "@material-ui/icons/ArrowBack";
+import Typography from "@material-ui/core/Typography";
+
+const useStyles = makeStyles(theme => ({
+  content: {
+    padding: "80px 10px 0px 10px"
+  },
+  returnDiv: {
+    display: "flex",
+    alignItems: "center",
+    marginBottom: "20px"
+  },
```

Comment:
is the width of this component the entire available screen? QuickRegister and the Completion page are both 66% vw, what do you think is a better approach?

Reply:
Good point, updated with the width of EventDetails set to 66% vw as well so its consistent with the related components.